test/nodejs/logging/logging.js

```diff
@@ -0,0 +1,189 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// This file is hand-written. Eventually the tests will be
+// replaced by auto-generated ones.
+
+var expect = require('chai').expect;
+var loggingV2 = require('@google-cloud/logging').v2;
+var storage = require('@google-cloud/storage');
+
+describe('loggingV2', function() {
+  var projectId = process.env.GCLOUD_TESTS_PROJECT_ID;
+
+  // It's unclear when the newly written entry appears in the list,
+  // so wait for a static time. TODO: should be consistent with
+  // other tests.
+  var WRITE_CONSISTENCY_DELAY_MS = 5000;
+
+  function wait(arg) {
+    return new Promise(function(r) {
+      setTimeout(function() { r(arg); }, WRITE_CONSISTENCY_DELAY_MS);
+    });
+  }
+
+  var opts = {projectId: projectId};
+  if (process.env.GCLOUD_TESTS_KEY) {
+    opts.keyFile = process.env.GCLOUD_TESTS_KEY;
+  }
+
+  describe('loggingServiceV2Client', function() {
+    var client;
+
+    beforeEach(function() {
+      client = loggingV2(opts).loggingServiceV2Client();
+    });
+
+    it('creates, lists, and deletes log entries', function(done) {
+      var logName = client.logPath(projectId, 'test');
+      var resourceName = client.parentPath(projectId);
+
+      var myLogMessage = 'My new test message';
+      var resource = {'type': 'global'};
+      var entries = [{textPayload: myLogMessage, logName: logName}];
+      var start = new Date();
+      client.writeLogEntries({
+        entries: entries,
+        resource: resource,
+        logName: logName
+      }).then(wait).then(function() {
+        return client.listLogEntries({
+          resourceNames: [resourceName],
+          filter: 'logName="' + logName + '"',
+          orderBy: 'timestamp desc'
+        });
+      }).then(function(resp) {
+        var entries = resp[0];
+        var found = false;
+        for (var i = 0; i < entries.length; ++i) {
+          var entry = entries[i];
+          if (entry.textPayload === myLogMessage && entry.logName === logName) {
+            found = true;
+            break;
+          }
+        }
+        expect(found).to.be.true;
+
+        return client.deleteLog({logName: logName});
+      }).then(function() {
+        done();
+      }).catch(done);
+    });
+
+    it('lists resource descriptors', function(done) {
+      client.listMonitoredResourceDescriptors().then(function(resp) {
+        expect(resp).not.to.be.empty;
```

Comment:
Add `expect(resp[0]).to.be.an('Array')`. To check that a list is being returned.